.github/workflows/deploy.yml

```diff
@@ -1,5 +1,8 @@
 name: Build and Deploy
-on: [push]
+on:
+  push:
+    branches:
+      - production
```

Comment:
Just to be clear: are we going to have a permanent production branch open? What will the git flow look like?

Merge to master, then merge to production?

Reply:
We can adjust as we learn more and find a better way, but my understanding is that we need a branch that the GitHub Actions watches, and when we merge to that branch (in this case production), Actions will run and automatically deploy to master. 

So it's master -- production -- develop -- feature. We'll merge features to develop, and develop to production and that's it. GitHub Action will merge production to master for us automatically which deploys the site. If I understand it all correctly that is! ;)